docs/source/markdown/podman-machine.1.md

```diff
@@ -0,0 +1,26 @@
+% podman-machine(1)
+
+## NAME
+podman\-machine - Manage Podman's virtual machine
+
+## SYNOPSIS
+**podman machine** *subcommand*
+
+## DESCRIPTION
+`podman machine` is a set of subcommands that manage Podman's virtual machine on MacOS.
+
+## SUBCOMMANDS
+
+| Command | Man Page                                                | Description                   |
+| ------- | ------------------------------------------------------- | ----------------------------- |
+| create  | [podman-machine-create(1)](podman-machine-create.1.md)  | Create a new virtual machine  |
+| remove | [podman-machine-destroy(1)](podman-machine-remove.1.md)| Remove a virtual machine     |
+| ssh     | [podman-machine-ssh.1.md(1)](podman-machine-ssh.1.md)   | SSH into a virtual machine    |
+| start   | [podman-machine-start(1)](podman-machine-start.1.md)    | Start a virtual machine       |
+| stop    | [podman-machine-stop(1)](podman-machine-stop.1.md)      | Stop a virtual machine        |
+
```

Comment:
It would be good to have a small User Guide like example here that shows how to start a VM, how to ssh into it, how to run a basic podman command on it, then remove the VM.